modules/subchannel/doc/content/source/scmclosures/SCMHTCGnielinski.md

```diff
@@ -0,0 +1,35 @@
+# SCMHTCGnielinski
+
```

Comment:
this correlation is for annular duct flow.
https://www.tandfonline.com/doi/epdf/10.1080/01457630802528661?needAccess=true
 
Has this been used for hexagonal ducts before?
What do the other codes use?

Reply:
This was introduced in https://github.com/idaholab/moose/pull/31547.
@tanoret thoughts?